docs/guide/dev/code_documentation/README.md

```diff
@@ -109,20 +109,20 @@ A minimal class documentation:
 
 When you want to reopen the class declaration in another file, use this:
 
-	/** @class {@linkapi CKEDITOR.editor CKEDITOR.editor} */
+	/** @class \{@linkapi CKEDITOR.editor CKEDITOR.editor\} */
 
 #### Details
 
 The order of tags may look strange, but you can remember it thanks to the following description:
 
-> Since 3.0 there is a private class `{@linkapi CKEDITOR.editor}` which extends `{@linkapi CKEDITOR.parent}` and mixins `{@linkapi CKEDITOR.event}` and `{@linkapi CKEDITOR.whatever}`.
+> Since 3.0 there is a private class `CKEDITOR.editor` which extends `CKEDITOR.parent` and mixins `CKEDITOR.event` and `CKEDITOR.whatever`.
 >
 > It has a private constructor (switched order &mdash; explained later in the tags list) which accepts the following parameters: ...
 
 Important tag details:
 
 * By default private classes will not be visible in the packages tree.
-* A good example of the difference between "mixins" and "extends" is that {@linkapi CKEDITOR.event CKEDITOR.event} is mixed in various other classes and the `CKEDITOR.dom.*` structure is based on extending parent classes.
+* A good example of the difference between "mixins" and "extends" is that {@linkapi CKEDITOR.event `CKEDITOR.event`} is mixed in various other classes and the `CKEDITOR.dom.*` structure is based on extending parent classes.
```

Comment:
Here also it should be a shrotcut version. Regarding backticks, it's a postprocessor role to properly wrap shortucted API links with `var` (or revelant semantic markup). By any means should it be handled in the content.